HerPublicWebsite/robots.txt

```diff
@@ -1,2 +1,9 @@
 User-agent: *
-Disallow: /portal/
\ No newline at end of file
+Allow: /questionnaire/
+Disallow: /portal/
+Disallow: /privacy-policy/
+Disallow: /accessibility-statement/
+Disallow: /digitalassistance/
+Disallow: /cookies/
+Disallow: /health-check/
+Disallow: /session-expired/
```

Comment:
I'd be more inclined for this to look something like
```Allow: /questionnaire/ (and anything else we do want allowed)
Disallow: /```

This takes the approach of `allow certain things but disallow everything else`

Reply:
Made these changes :D